src/Layout/components/Header.tsx

```diff
@@ -35,11 +39,11 @@ const Header = () => {
               <span onClick={handleLogout} className="cursor-pointer">
                 Logout
               </span>
-              <NavLink to="/signin">Log in</NavLink>
+              {userLoggedIn ? "" : <NavLink to="/signin">Log in</NavLink>}
```

Comment:
Why the ternary comparison here. Use the && shortciruit

Reply:
I have just resolved it